functions/source/update_custommanaged_cmk_policy.py

```diff
@@ -0,0 +1,167 @@
+import json
+import logging
+import threading
+import boto3
+import cfnresponse
+
+# boto client
+
+client = boto3.client('kms')
+
+databricks_s3bucketonly_statement1 = {
+  "Sid": "Allow Databricks to use KMS key for DBFS",
+  "Effect": "Allow",
+  "Principal":{
+    "AWS":"arn:aws:iam::414351767826:root"
+  },
+  "Action": [
+    "kms:Encrypt",
+    "kms:Decrypt",
+    "kms:ReEncrypt*",
+    "kms:GenerateDataKey*",
+    "kms:DescribeKey"
+  ],
+  "Resource": "*"
+}
+
+databricks_s3bucketonly_statement2 = {
+  "Sid": "Allow Databricks to use KMS key for DBFS (Grants)",
+  "Effect": "Allow",
+  "Principal":{
+    "AWS":"arn:aws:iam::414351767826:root"
+  },
+  "Action": [
+    "kms:CreateGrant",
+    "kms:ListGrants",
+    "kms:RevokeGrant"
+  ],
+  "Resource": "*",
```

Comment:
Same. Can we restrict to specific resources? Please do the same for ever policy where we have mentioned Resource: *

Reply:
same comment applies